src/components/modals/DonateModal.tsx

```diff
@@ -34,6 +34,8 @@ export interface IDonateModalProps extends IModal {
 	anonymous?: boolean;
 	setSuccessDonation: (i: ISuccessDonation) => void;
 	givBackEligible?: boolean;
+	mainProjectAddress?: string;
+	secondaryProjectAdress?: string;
```

Comment:
we never use these addresses in the donate modal. I think we can remove them.

Reply:
same as above